pimd/pim_igmpv2.c

```diff
@@ -127,6 +127,13 @@ int igmp_v2_recv_report(struct gm_sock *igmp, struct in_addr from,
 
 	memcpy(&group_addr, igmp_msg + 4, sizeof(struct in_addr));
 
+	if (to.s_addr != group_addr.s_addr) {
+		if (PIM_DEBUG_GM_PACKETS)
+			zlog_debug(
+				"IGMPv2 report message is ignored since received on address other than Group-address");
```

Comment:
Would be good to ad the interface and the addresses in question to the debug message.

Reply:
Done.

```
2025/07/16 21:20:47 PIM: [JXRJG-SR17W] IGMPv2 report message from 88.8.8.8 on enp1s0 is ignored since received on address other than Group-address(239.1.2.4)
```